### ISSUE TITLE
CBO/Org Onboarding Registration (Claim a record workflow #5)

### ISSUE THREAD

**Issue by EnriqueCorado** [benetech/ServiceNet#1326]
CBO/Org Onboarding Registration (Claim a record workflow #4)

**Comment by johnhbenetech** [comment]
see comment on #1323 - the top of the home page will already display their claimed records, so should we just continue to use that?

**Comment by EnriqueCorado** [comment]
@johnhbenetech 

yes, will merge 1323 into this one with claim more records under 'have more data to contribute'
